oracle/unlock_sample_data.sql

```diff
@@ -0,0 +1,7 @@
+alter user hr account unlock;
+/
+
+alter user hr identified by oracle;
```

Comment:
Updated in commit https://github.com/OraOpenSource/oraclexe-apex/commit/b072e6575fcecca0d1808bcfb570e8703d247589